src/components/PDFThumbnail/index.tsx

```diff
@@ -31,10 +33,13 @@ function PDFThumbnail({previewSourceURL, style, isAuthTokenRequired = false, ena
                     onPassword();
                 }}
             >
-                <Thumbnail pageIndex={0} />
+                <Thumbnail
+                    pageIndex={0}
+                    className={isClickable ? '' : 'react-pdf__Thumbnail--notClickable'}
```

Comment:
Instead of using `className`, using a `View` with `pointerEvents=none`

Reply:
Great! This method works perfectly for removing the default pointer style added by the browser to hyperlinks.